### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by sahil-shubham**
## Changes
- added basic info about credential security; 
- stated that this is not an official app by IIT KGP

### CODE REVIEW

**Comment by sahil-shubham** [comment]
@thealphadollar I preferred adding the links to the information regarding `localStorage` and where the credentials are stored, rather than explaining those in the README. This way we can keep the README file less cluttered. 

What do you think about this? And how would you have approached this task?

File: README.md [hunk 1]
```diff
@@ -2,7 +2,14 @@ IITKGP ERP AUTO LOGIN
 =====================
 A browser extension that has to be set up just one time for automatic login to your IITKGP ERP account every time. Just [install the chrome extension](https://chrome.google.com/webstore/detail/iitkgp-erp-auto-login/gdickphilmdekcbmpjmbnbikchaecbdk) or [install the firefox addon](https://addons.mozilla.org/addon/iitkgp-erp-auto-login/), and click on the extension logo (IIT Kharagpur's logo) to set up your login credentials.
 
-<img src="instruction_images/demo.png">
+   <img src="instruction_images/demo.png" />
+
+**Note:- This is not an official application by IIT Kharagpur. This is the result of hardwork and enthusiasm by [Metakgp](https://metakgp.github.io/).**
```
Comment (thealphadollar) [inline README.md:7]:
```suggestion
**Note:- This is not an official application by IIT Kharagpur. This is the result of hard work and enthusiasm by @athityakumar and is maintained by [Metakgp](https://metakgp.github.io/).**
```

**Review by thealphadollar — APPROVED** [review]
Yeah! This is really cool and good to let people know this is not by the administration. However, I would just add Athitya as the original developer.

I also liked the minimalism you have adopted towards revealing only the information required. Since most people will not be interested (and may lack the knowledge) to understand the details, it is good to have resources where they can look at if they want and let it be concise otherwise.

Thank you very much for your contributions @sahil-shubham :sparkles:

**Comment by sahil-shubham** [comment]
I have added the suggested changes, it was a good decision to add the original developer. Thanks @thealphadollar

**Review by thealphadollar — APPROVED** [review]

**Pull request merged**